### PR TITLE
 #61 - Create a Network switch component

### LINKED ISSUE #61: Create network switch component
## Description:

Allow the user to update the wallet network in the settings screen. Show the current network in the wallet overview screen.

### PR DESCRIPTION
**Pull request by Koushith**
## Description

- [x] Adds Network switch option in advanced settings.
- [x] Removed modal from the top bar and replaced it with a custom dropdown menu component.
- [x] Icon support in Badge component

## Screenshot

https://www.loom.com/share/f2126c6dae57479dbedb568b33116858
https://www.loom.com/share/5a4ec912b41044d791abb2a7aba2575d

### CODE REVIEW

File: src/components/primitive/drop-down-menu/dropdown-menu.component.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+export const DropDownMenu = (props: any) => {
+  const [isActive, setIsActive] = useState(false);
+
+  const { icon, children } = props;
+
+  return (
+    <Box>
+      <Box>
+        <IconContainer onClick={() => setIsActive(!isActive)}>
```
Comment (koshikraj) [inline src/components/primitive/drop-down-menu/dropdown-menu.component.tsx]: We should make the drop down container inactive when clicked elsewhere on the screen too.

File: src/screens/wallet-settings/wallet-settings.screen.tsx [hunk 2]
```diff
@@ -39,20 +58,26 @@ export const WalletSettingsScreen = ({ history }: any) => {
...
+            <Box row hCenter marginTop={2} justify={'between'}>
+              <Label>Signaling Period</Label>
+              <SignnalingInput type='text' placeholder='10 days' />
+            </Box>
+            <Box marginTop={2}>
+              <DropDown
+                placeholder='select network'
+                label='Select Network'
+                options={networkOptions}
+                onChange={(e: any) => setSelectNetwork(e)}
```
Comment (koshikraj) [inline src/screens/wallet-settings/wallet-settings.screen.tsx:77]: The font of the option once selected is small and different from the default. Can you check this?

Comment (Koushith) [reply src/screens/wallet-settings/wallet-settings.screen.tsx:77]: oh yeah. working on it.

**Comment by Koushith** [comment]
image


 - [x] Fixes Dropdown font.
 - [x] unmount component when clicked outside.

**Review by koshikraj — APPROVED** [review]

**Pull request merged**